### PR TITLE
Board

### PR DESCRIPTION
**Pull request by zony249**

### CODE REVIEW

File: project3/connect4_frontend/src/main.rs [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+use fsm::*;
+
+mod boardgame;
+use boardgame::*;
+
+mod chip;
+use chip::*;
+
+mod ai;
+use ai::*;
```
Comment (dimassudjito) [inline project3/connect4_frontend/src/main.rs:63]: is there an ai.rs file that I'm missing?

**Review by dimassudjito — COMMENTED** [review]
Couldn't compile

File: project3/connect4_frontend/src/main.rs (hunk 1 above)
Comment (zony249) [reply project3/connect4_frontend/src/main.rs:63]: Just added it. I indeed forgot to commit the ai.rs file

**Review by dimassudjito — COMMENTED** [review]
There's a merge conflict now after Josh's and my PR. Also, would it be possible to have a bit of description on how to run the CLI program for the board. Thanks!

**Comment by zony249** [comment]
Ok, I'll add a description later maybe. This CLI code is meant to be for debug purposes only, I'll either stash it into a function somewhere or delete it altogether once I'm done

**Pull request merged**